### ISSUE TITLE
Santorini Theme minimum contrast adjustments

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13499]
**Describe the bug**

- Automated test detects contrast below standard.
- Cart page seems to gather most of the contrast issues, so this is a good page for testing.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Add a couple of items to the cart and go to Cart page.
2. Inspect reported elements (screenshot below) 
3. Go to the "Computed" tab (next to "Styles") to confirm font-size
4. Use an online WCAG Contrast Checker to verify current contrast.

**Expected behavior**

- Pass automated test.
- Text less than 18 points or bold text less than 14 points must have a luminosity contrast ratio of **4.5:1 or more**.
- Text 18 point or larger or bold text 14 point or larger must have a luminosity contrast ratio of **3:1 or more**.
- When color alone is used to indicate links from non-linked text - the color difference between the surrounding text must meet these contrast requirements.
- It seems the automated validator takes 18pt sharp under the "less than 18pt" threshold, therefore any text of 18pt or less should have a contrast ratio of **4.5:1 or more**.

**Screenshots**

![Screen Shot 2021-08-18 at 3 19 39 PM](https://user-images.githubusercontent.com/15113219/129988081-91964be8-6b5a-4ffa-826f-beaca50c1057.png)
![Screen Shot 2021-08-18 at 3 20 27 PM](https://user-images.githubusercontent.com/15113219/129988084-294035d5-bb0d-4bf3-9be7-f380b3d5aeea.png)
![Screen Shot 2021-08-18 at 3 22 39 PM](https://user-images.githubusercontent.com/15113219/129988085-ae40297c-cc38-442c-be77-16b04aa89afc.png)
![Screen Shot 2021-08-18 at 8 03 03 PM](https://user-images.githubusercontent.com/15113219/130704955-02e719eb-23e1-4cc0-b23b-41be92fab2da.png)

**Comment by saad-mo** [comment]
Most of these issues can be addressed via styles. assigning to @morganm58

**Comment by Manurod2018** [comment]
For AA compliance:
Changed “Main / Link” color from #2480C5 to #1672B7
Changed “Alert / Success” color from #5DAC06 to #38871F
